htmlgenerator/base.py

```diff
@@ -34,7 +37,9 @@ def __init__(self, *children):
                 ] = f"{frame.filename}:{frame.lineno} in {frame.function}"
             self._src_location = (frame.filename, frame.lineno, frame.function)
 
-    def _try_render(self, element, context):
+    def _try_render(
+        self, element: typing.Any, context: dict
```

Comment:
```suggestion
        self, element: Any, context: dict
```
is the `typing.` necessary? Can we not import Any directly? I think everyone should know that `Any` comes from `typing`.

Reply:
I hope you are aware that this has potential for a programmers religious war :rofl: 

If you know C++ a bit you might have heard about that ```using namespace std```-controversy. Many people there argue that you should write

    std::cout < "Hello world";

instead of 

    using namespace std;
    cout < "Hello world";

but many people prefer the shorter version (which is understandable because C++ is so verbouse).

As far as I remember in Java you amost never prefix namespaces, right?
In Python both ways are acceptable. I think the best way to decide is to determine how we want to communicate a dependency to module/package or a dependency to a certain name inside a module/package. If we always specify all names we plan to use then the import list on the top can get very verbose. If we always only specify the module or package then it will be harder to understand the actual dependencies of our module.
I would keep ```typing``` for now because type hints are very "unadapted" in the Python world (even my PR uses the word "explore" ;). For now it is good to make it very clear that we are using type hints here.